src/screens/wallet-overview/components/wallet-activities.component.tsx

```diff
@@ -24,36 +27,29 @@ export const StyledDiv = styled.div`
   min-width: 30rem;
 `;
 
-export const AllActivities = () => {
+export const AllActivities: React.FC<AllActivitiesProps> = (props) => {
+
+  const { transactions } = props;
+
   return (
     <Box hCenter vCenter>
       <Text variant='small' tx='walletOverViewPage.activities' color='textLight' bold600 />
-      {/* map this */}
-      <Activities>
-        <Box flex={0}>
-          <IconSvg name='recieved' size='xLarge' />
-        </Box>
-        <StyledDiv>
-          <Text variant='small' text='Recieved from 0x70997970C51812dc..' color='textLight' />
-          <Text variant='small' text='2 days ago' color='textLighter' />
-        </StyledDiv>
-        <Box>
-          <Text variant='small' text='0.1 ETH' color='textLighter' />
-        </Box>
-      </Activities>
 
-      <Activities>
+      {
+        transactions.map((transaction) => 
+        <Activities>
         <Box flex={0}>
-          <IconSvg name='sent' size='xLarge' />
+          <IconSvg name={getActivityInfo(transaction.event).iconName} size='xLarge' />
         </Box>
         <StyledDiv>
-          <Text variant='small' text='Sent to 0x70997970C51812dc..' color='textLight' />
-          <Text variant='small' text='2 days ago' color='textLighter' />
+          <Text variant='small' text={`${getActivityInfo(transaction.event).text} ${AddressUtil.shorternAddress(transaction.address)}`} color='textLight' />
+          <Text variant='small' text={transaction.age + ' ago'} color='textLighter' />
```

Comment:
can we use ```ceil()``` here? 
![image](https://user-images.githubusercontent.com/30016242/157616995-01a4993f-e5f2-4812-9b64-968c9a590a74.png)

Reply:
Yeah, this has been fixed later. Have added floor.